src/components/nav/sidebarItem.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react'
+import { FC } from 'react'
+import * as Sidebar from './style'
+import { SidebarLink } from './sidebarLink'
+import { Icon } from 'react-icons-kit'
+
+const genres = [
+  'action',
+  'adventure',
+  'animation',
+  'comedy',
+  'horror',
+  'family',
+  'crime',
+  'disaster',
+  'documentary',
+  'drama',
+  'eastern',
+  'fan-film',
+  'fantasy',
+  'film-noir',
+  'history',
+  'holiday',
+]
+
+
+export const SidebarItemGroup: FC<{
+  toggleCategories(): void,
+  size: number,
+  icon: any,
```

Comment:
Can you find a substitute for `any`?

Reply:
It's defined as `any` in the module I've imported it from.